GliaWidgets/Coordinator/RootCoordinator.Environment.swift

```diff
@@ -0,0 +1,23 @@
+extension RootCoordinator {
+    struct Environment {
+        var chatStorage: Glia.Environment.ChatStorage
+        var fetchFile: CoreSdkClient.FetchFile
+        var sendSelectedOptionValue: CoreSdkClient.SendSelectedOptionValue
+        var uploadFileToEngagement: CoreSdkClient.UploadFileToEngagement
+        var audioSession: Glia.Environment.AudioSession
+        var uuid: () -> UUID
+        var fileManager: FoundationBased.FileManager
+        var data: FoundationBased.Data
+        var date: () -> Date
```

Comment:
Maybe this one as well. I guess it's the current date.

Reply:
I guess in order to clarify this I need to provide a bit of context about Environment(s) that are widely used in Widgets SDK. Basically Environment is a dependency container that solves the problem exchanging dependencies during testing. 
So regarding particular naming of things like `var date: () -> Date` and `var uuid: () -> UUID` is way of saying is that there is going to be used `Date()` and UUID() accordingly. In case of using some type that has several initializers that expect specific arguments, there will be used wrapper struct instead of simple closure, like `var  data: FoundationBased.Data` fro example. But the idea stays the same - for using 3rd party frameworks like CoreSDK and 1st party like `Foundation.FileManager` we will have closures and structs-wrappers that mimic actual types provided by frameworks. 
Child Environment that is going to use `Foundation.Date` for some very specific case will get this type injected from parent Environment that may not use it explicitly (or use it for some very different case), so there's no sense give to parent Environment property specific name, but very general one like `var date: () -> Date`. Does it make more sense now?

Reply:
All of it makes sense :) but maybe if you add something like this in the code, then it would be perfect, and then the names can stay as is.

Reply:
Added documentation regarding Environment(s) in the framework and naming conventions 8f8892caf7aa9b548a6619e7a253524f4a858137.